agent/handlers/gql/schema.go

```diff
@@ -0,0 +1,41 @@
+package gql
+
+import (
+	"github.com/aws/amazon-ecs-agent/agent/api"
+	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+	"github.com/aws/amazon-ecs-agent/agent/handlers/utils"
+	v3 "github.com/aws/amazon-ecs-agent/agent/handlers/v3"
+	"github.com/aws/amazon-ecs-agent/agent/stats"
+	"github.com/cihub/seelog"
+	"github.com/graphql-go/graphql"
+)
+
+var ContainerMetadataPath = "/graphql/" + utils.ConstructMuxVar(v3.V3EndpointIDMuxName, utils.AnythingButSlashRegEx)
```

Comment:
let's make this variable unexported

Reply:
Should it be defined in `task_server_setup` because it is used in that file. I was trying to follow the format of the other endpoints.